src/common/header/components/GitHubButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { IconButton } from '@chakra-ui/button';
+import { AiFillGithub } from 'react-icons/ai';
+
+export function GitHubButton(): JSX.Element {
+  return (
+    <IconButton
+      as="a"
+      href="https://github.com/VikeLabs/courseup"
+      target="_blank"
+      colorScheme="none"
+      fontSize="35px"
```

Comment:
This is making the header bigger than the overall contents btw.

Reply:
sorry, totally missed that! 🤦🏼‍♂️ I've [committed a change that fixes that issue now](https://github.com/VikeLabs/courseup/pull/257/commits/7eda0d8a456e9532697cc91c39bbab55985463fa)!